lib/vanagon/component.rb

```diff
@@ -305,7 +305,7 @@ def deprecated_rewrite_url
     def get_source(workdir) # rubocop:disable Metrics/AbcSize, Metrics/PerceivedComplexity
       opts = options.merge({ workdir: workdir })
       if url || !mirrors.empty?
-        if ENV['VANAGON_USE_MIRRORS'] == 'n'
+        if ENV['VANAGON_USE_MIRRORS'] == 'n' or ENV['VANAGON_USE_MIRRORS'] == false
```

Comment:
`false` needs to be quoted as environment variables are strings.

Reply:
Maybe even?

```ruby
%w[ n no false ].include? ENV['VANAGON_USE_MIRRORS'].downcase
```

Reply:
Thanks @Sharpie. I was banging my head on that test failure for a bit. 

@e-gris That could be added. It involved a few more if statements since you can't downcase a nil object and show you have to check for that first. Not sure how important this is overall. It's not hard to add, but I also don't want to make this more complicated to follow for any future maintainers :)